mp/src/game/client/momentum/ui/controls/FileImage.cpp

```diff
@@ -18,6 +18,52 @@
 
 using namespace vgui;
 
+struct ImageCacheEntry
+{
+    CUtlBuffer m_bufOriginalImage;
+};
+
+class CFileImageCache
+{
+public:
+    CFileImageCache() {}
+
+    void AddImageToCache(const char *pPath, const CUtlBuffer &pBuf)
+    {
+        const auto pEntry = new ImageCacheEntry;
```

Comment:
Might be worth adding an assert to make sure we're not adding duplicate image paths

Reply:
Resolved in https://github.com/momentum-mod/game/pull/754/commits/d00e94b6cce6e87aaafb191a774abb016e74943e